init.sh

```diff
@@ -783,9 +766,8 @@ esac
 done
 
 if [[ -z "$action" ]]; then
-  MSG="Usage: $script_name [OPTION]\nTry '$script_name --help' for more information."
-  echo -e "$MSG"
-  exit 0
+  MSG="No action specified!"
+  print_message "error" "$MSG"
```

Comment:
We need to print a helpful message and stop the script when no option is used as some Unix commands already do.